### PR TITLE
Integrating smaller lantern-ui into flashlight-build

### PR DESCRIPTION
**Pull request by oxtoacart**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,6 +59,11 @@ The script `tagandbuild.bash` tags and runs crosscompile.bash.
 
 Note - ./crosscompile.bash omits debug symbols to keep the build smaller.
 
+### Linux
+
+crosscompile currently can't target Linux, so you will need to manually build on
```
Comment (xiam) [inline README.md:64]: @oxtoacart  If this is a known bug or limitation, could you please add a reference URL, an error message or a very short explanation on the reason?

**Comment by xiam** [comment]
Code looks fine and ready to merge, I've just added a tiny note on the documentation.

File: README.md (hunk 1 above)
Comment (oxtoacart) [inline README.md:64]: Fixed.

**Comment by oxtoacart** [comment]
Thanks for the review!

**Pull request merged**